src/Factories/WriterFactory.php

```diff
@@ -32,7 +32,7 @@ public static function make($type, $export): WriterInterface
             return static::makeCsvWriter($type, $export);
         }
 
-        return SpoutWriterFactory::createFromType($type);
+        return SpoutWriterFactory::createWriter($type);
```

Comment:
This is OK, but doesn't improve anything, it still uses `Box\Spout\Writer\Common\Creator\WriterFactory::createFromType` behind the scene